.github/workflows/ci.yml

```diff
@@ -251,6 +251,10 @@ jobs:
         timeout-minutes: 5
         run: |
           make RELEASE=true UNINSTALL=noclean smoke-test-tls-root
+      - name: Test examples
+        run: |
+          examples/tests/test-simple.sh
+          examples/tests/test-echo.sh
```

Comment:
add timeout parameters to limit runaway tasks

Reply:
Hmm I tried adding timeouts to the `test-simple.sh` script and I tried adding timeouts directly to the `echo` example... I'm trying to figure out what's going on

Reply:
add `timeout-minutes` in the GH Action run which terminate task if exceeds allotted time.  See previous run